src/bindings/io/uv/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+include(LibAddMacros)
+
+find_package (PkgConfig)
+pkg_check_modules (LIBUV QUIET libuv)
+
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+	remove_binding(io_uv "cannot be built on Windows")
+elseif(NOT LIBUV_FOUND)
```

Comment:
Spaces before "(". (Yes, some older CMake code does this still wrongly, fixes are welcomed.)

Reply:
I'm not sure: Is the rule space or no space before "("?

Reply:
There should be a space before "(".

Rationale: We do not have detailed guidelines for CMake but in case of doubt we use the guidelines we already have for C.